css/home.css

```diff
@@ -67,6 +67,17 @@
   height: 200px;
```

Comment:
can use rem here

Reply:
Yes, but I want the figures to be specifically 200px and not dependent on the root font size.